### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by grdumas**
List each helper script in README and briefly describe its functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+Helper scripts to aid the execution and post-processing of uperf network performance tool.
+
+## Scripts
+
+Name | Description
+-----|------------
+uperf-base | Checks if the bench-base library can be sourced from the ${TOOLBOX_HOME}/bash/library/ directory. If the library cannot be sourced, the script prints an error message and exits with an exit status of 1.
+uperf-client | Runs a network benchmark using uperf, a network performance tool. The script has several command-line options that can be used to customize the benchmark run, including the number of threads, protocol, packet sizes, and duration.
+uperf-get-runtime | Outputs the duration of a benchmark run.
+uperf-post-process | Perl script that processes results from the uPerf benchmark tool and outputs a JSON file containing performance metrics.
```
Comment (k-rister) [inline README.md:11]: Let's go ahead and reference that these JSON documents adhere to the CDM (CommonDataModel) and include a link to that repository.

**Review by k-rister — CHANGES_REQUESTED** [review]
This is good stuff, lets just add a little bit more.

File: README.md (hunk 1 above)
Comment (k-rister) [reply README.md:11]: Except that it is our CDM repo, not microsoft's :)

https://github.com/perftool-incubator/CommonDataModel

**Review by k-rister — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (grdumas) [reply README.md:11]: Fixed!

**Review by k-rister — APPROVED** [review]

**Pull request merged**